appveyor_build.sh

```diff
@@ -41,33 +43,34 @@ function set_configuration {
     echo "Edit $FILE to set PREFIX=$2"
     sed -e "/PREFIX=/s|=.*|=$2|" \
         -e "/^ *CFLAGS *=/s/\r\?$/ $3\0/" \
-         config/Makefile.$1 > config/Makefile
+         "config/Makefile.$1" > config/Makefile
 #    run "Content of $FILE" cat config/Makefile
 }
 
-APPVEYOR_BUILD_FOLDER=$(echo $APPVEYOR_BUILD_FOLDER| cygpath -f -)
+APPVEYOR_BUILD_FOLDER=$(echo "$APPVEYOR_BUILD_FOLDER" | cygpath -f -)
 # These directory names are specified here, because getting UTF-8 correctly
 # through appveyor.yml -> Command Script -> Bash is quite painful...
-OCAMLROOT=$(echo $PROGRAMFILES/Бактріан🐫| cygpath -f - -m)
+OCAMLROOT=$(echo "$PROGRAMFILES/Бактріан🐫" | cygpath -f - -m)
 
 # This must be kept in sync with appveyor_build.cmd
 BUILD_PREFIX=🐫реализация
 
-export PATH=$(echo $OCAMLROOT| cygpath -f -)/bin/flexdll:$PATH
+PATH=$(echo "$OCAMLROOT" | cygpath -f -)/bin/flexdll:$PATH
+export PATH
```

Comment:
@stedolan is correct, the `export` is in fact completely unnecessary (see [Posix](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_05_03)).

Reply:
I'm OK with removing it. Can you point to the relevant part of your link (I try to find it but I failed as there's a lot of thing in it...) ? Just out of curiosity.

Reply:
It's the sentence "If a variable is initialized from the environment, it shall be marked for export immediately" at http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_05_03

You can see this if you have two scripts `foo` and `bar`:

`foo`:
```sh
#!/bin/sh
FOO='Updated FOO'
./bar
```
`bar`:
```sh
#!/bin/sh
echo "FOO=$FOO"
```
<pre><font color="#8AE234"><b>dra@cosmic</b></font>:<font color="#729FCF"><b>~</b></font>$ ./foo
FOO=
<font color="#8AE234"><b>dra@cosmic</b></font>:<font color="#729FCF"><b>~</b></font>$ FOO=foo ./foo
FOO=Updated FOO
</pre>